cmake/BuildDocumentation.cmake

```diff
@@ -7,7 +7,7 @@
 # this will get more complicated as we add things like bibtex and makeindex, for now just copy the files into the build tree, execute xelatex twice and rename the output
 #message("Hoping to copy from ${ORIGINAL_CMAKE_SOURCE_DIR}/doc/${INNAME} to ${ORIGINAL_CMAKE_BINARY_DIR}/doc-build/${INNAME}")
 execute_process( COMMAND cmake -E copy_directory ${ORIGINAL_CMAKE_SOURCE_DIR}/doc/${INNAME} ${ORIGINAL_CMAKE_BINARY_DIR}/doc-build/${INNAME})
-execute_process( COMMAND ${XELATEX} -interaction=nonstopmode ${INNAME}.tex )
-execute_process( COMMAND ${XELATEX} -interaction=nonstopmode ${INNAME}.tex )
-execute_process( COMMAND ${XELATEX} -interaction=nonstopmode ${INNAME}.tex )
+execute_process( COMMAND ${XELATEX} -interaction=${TEX_INTERACTION} ${INNAME}.tex )
```

Comment:
This is now no longer hardcoded, but instead relies on the caller to set the value